### PR TITLE
Lists and toast component

### PR DESCRIPTION
**Pull request by zoyatanweer**
- added ordered, unordered and stacked list.
-  added components in the side navigation bar.
- added toast component.

image
image
image

### CODE REVIEW

File: components/toast/toast.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+}
+.toast-para{
+    font-size: 2rem;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+}
+
+.toast-svg{
```
Comment (dev-prabhat) [inline components/toast/toast.css:28]: irregular spacing between classes

**Review by dev-prabhat — CHANGES_REQUESTED** [review]
Components are clean and looks great   Code is clean, readable and just some irregular spacing but CSS classes

File: components/lists/lists.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
+.lists{
+    border: 2px solid var(--primary-color);
+    margin-bottom: 2rem;
+}
+.lists-divider{
+    background-color: var(--alert-secondary-color);
+    height: 1px;
+    max-width: 80%;
+    margin-left: 17px;
```
Comment (srishtigautam21) [inline components/lists/lists.css:9]: you can use rem here

**Review by srishtigautam21 — APPROVED** [review]
looks good to merge with the suggested changes
Appreciation:The UI looks very clean

File: components/toast/toast.css (hunk 1 above)
Comment (zoyatanweer) [reply components/toast/toast.css:28]: ok i will remove that

File: components/lists/lists.css (hunk 2 above)
Comment (zoyatanweer) [reply components/lists/lists.css:9]: sure srishti

**Comment by zoyatanweer** [comment]
thank you Prabhat and Srishti for your reviews.

**Review by zoef-yt — APPROVED** [review]
The code looks good to me with all the changes.
1. The UI looks really good.
2. The code is simple and understandable
3. Nice use of comments to show the components starts.

Good to merge. Good work.

**Review by srishtigautam21 — APPROVED** [review]
clean and understandable code
good to merge

**Review by itsmebipul00 — APPROVED** [review]
Good to merge

**Pull request merged**